### DISCUSSION TITLE
Can you import presets to the plugins version of NTSC-RS?

### DISCUSSION THREAD

**Discussion by DeeheNoughs** [ntsc-rs/ntsc-rs#174] [Q&A] [answered]
I wanna for example, import a specific preset I adjusted in the NTSC-RS app to the plugin in after effects, is that possible or do I have to just copy it manually?

**Comment by ghost** [comment]
Copy manually

**Comment by valadaptive** [comment, marked as answer]
This was added a bit ago, in v0.9.0!
